win32/config_sh.PL

```diff
@@ -148,6 +148,12 @@ my $ver = `ver 2>nul`;
 $opt{osvers} = $ver =~ /\b(\d+(?:\.\d+)+)\b/ ? $1 : '4.0';
 
 if (exists $opt{cc}) {
+    `$opt{cc} -o have_stdckdint.exe configure/have_stdckdint.c 1>nul 2>&1`;
```

Comment:
I don't like seeing backticks in void context. Why isn't this just
```suggestion
    system "$opt{cc} -o have_stdckdint.exe configure/have_stdckdint.c 1>nul 2>&1";
```
?